CMakeLists.txt

```diff
@@ -53,7 +53,7 @@ endif()
 
 if(CMAKE_COMPILER_IS_CLANG)
   # Fix for Qt5 - using namespace QtDeclarative in header
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-header-hygiene")
+  set(COMMON_CXX_FLAGS "${COMMON_CXX_FLAGS} -Wno-header-hygiene")
```

Comment:
Is there a need for this change? I would prefer to keep CMAKE_CXX_FLAGS. COMMON_CXX_FLAGS is more like an implementation detail of how common_compiler_flags() works IMHO...